docs/src/languages/php/tuning.md

```diff
@@ -171,48 +205,51 @@ Remember to remove the `cachetools.phar` file once you're done with it.
-If you have disabled OPcache timestamp validation,
-you need to clear the OPcache explicitly on deployment (which can be done by restarting PHP-FPM).
+By default, OPcache checks that the cached version of a file is always up to date.
+It means that every time a cached file is used, OPcache compares it to the file on disk.
+If that file has changed it gets reloaded and recached.
 
-Note that some applications generate PHP code at runtime based on user configuration.
-If your application does that,
-you can't disable the timestamp validation
-as it would prevent updates to the generated code from being loaded.
+If you know your code isn't going to change outside of a new deploy,
+you can disable that check and get a small performance improvement.
 
-The timestamp validation may be disabled with an `ini` setting,
-and the easiest way to do so is via `.platform.app.yaml`:
+The timestamp validation can be disabled with:
 
-```yaml
+```yaml {location=".platform.app.yaml"}
 variables:
     php:
         'opcache.validate_timestamps': 0
 ```
 
+When you have disabled OPcache timestamp validation,
+you need to explicitly clear OPcache on deployment by restarting PHP-FPM.
+
+Note: If your application generates PHP code at runtime based on user configuration, you can't disable the timestamp validation.
+Doing so would prevent updates to the generated code from being loaded.
+
 ## Optimize your code
 
-It's also possible that your own code is doing more work than it needs to.
-Profiling and optimizing a PHP application is a much larger topic than fits here,
-but Platform.sh recommends enabling [Blackfire.io](../../increase-observability/integrate-observability/blackfire.md)
+It's possible that your application is doing more work than it needs to.
+Profiling and optimizing a PHP application is a much larger topic than fits here.
+You can use [Blackfire.io](../../increase-observability/integrate-observability/blackfire.md)
 on your project to determine what slow spots can be found and addressed.
 
-The web agency [Pixelant](https://www.pixelant.net/) has also published a [log analyzer tool for Platform.sh](https://github.com/pixelant/platformsh-analytics).
-It works only for PHP scripts,
-but offers good visualizations and insights into the operation of your site
-that can suggest places to further optimize your configuration
-and provide guidance on when it's time to increase your plan size.
-(Note that this tool is maintained by a 3rd party, not by Platform.sh.)
+The web agency [Pixelant](https://www.pixelant.net/) has published a [log analyzer tool for Platform.sh](https://github.com/pixelant/platformsh-analytics).
+
+This tool works only for PHP scripts,
+but offers good visualizations and insights into the operation of your site.
+
+It suggests where to further optimize your configuration
+and provides guidance on when to increase your plan size.
+
+Note that this tool is maintained by a 3rd party, not by Platform.sh.
```

Comment:
:hammer_and_wrench: I would maybe get rid of this section. It doesn't seem to rise to the level of documentation (it's not even certain it works outside of macOS).